ophyd/device.py

```diff
@@ -1027,7 +1027,7 @@ def stop(self, *, success=False):
                                  for sub_attr, ex in ex.exceptions.items()])
             except Exception as ex:
                 exc_list.append((attr, ex))
-                logger.error('Device %s (%s) stop failed', attr, dev,
+                self.log.error('Device %s (%s) stop failed', attr, dev,
```

Comment:
Since you're touching these lines, mind switching `log.error(..., exc_info=ex)` -> `log.exception`?